platform/linux-generic/odp_ipsec_sad.c

```diff
@@ -900,3 +908,18 @@ uint16_t _odp_ipsec_sa_alloc_ipv4_id(ipsec_sa_t *ipsec_sa)
 	/* No need to convert to BE: ID just should not be duplicated */
 	return tl->next_ipv4_id++;
 }
+
+uint64_t _odp_ipsec_sa_stats_pkts(ipsec_sa_t *sa)
+{
+	uint64_t per_thread_pkt_cnt = 0;
```

Comment:
Please rename the variable as it is not a per-thread count. Actually, it is not really a packet count either, but the number of future packets already added to the per-SA packet count but not yet processed.

The comment line below ("aggregate per thread packet counts") may also mislead the casual reader so I would suggest either just removing the comment or rephrasing it.

Reply:
Will remove the comment. As for variable name, do you have any suggestions? 

My suggestion: 'tl_pkt_quota'

Reply:
I do not have a good suggestion (maybe precounted_packets?). I would not complain about tl_pkt_quota.